backend/open_webui/utils/filter.py

```
@@ -1,34 +1,59 @@
 import inspect
 import logging
 
-from open_webui.utils.plugin import load_function_module_by_id
+from open_webui.utils.plugin import (
+    load_function_module_by_id,
```

Ruff (0.12.2)

`open_webui.utils.plugin.load_function_module_by_id` imported but unused

Remove unused import: `open_webui.utils.plugin.load_function_module_by_id`

(F401)